Working_Code/requirements_pyenv.txt

```diff
@@ -0,0 +1,9 @@
+pandas
+requests
+openpyxl
+scikit-learn~=1.5.2
```

Comment:
This version of sci-kit learn is stable for random impute testing because of a bug that 1.6 and 1.7 have: https://github.com/skorch-dev/skorch/issues/1079